src/test/suite/ModuleResolver.test.ts

```diff
@@ -0,0 +1,91 @@
+import * as assert from "assert";
+import * as path from "path";
+import * as prettier from "prettier";
+// tslint:disable-next-line: no-implicit-dependencies
+import * as sinon from "sinon";
+
+import { getWorkspaceFolderUri } from "./format.test";
+import { ModuleResolver } from "../../ModuleResolver";
+import { LoggingService } from "../../LoggingService";
+import { NotificationService } from "../../NotificationService";
+
+suite("Test ModuleResolver", function () {
+  let moduleResolver: ModuleResolver;
+  let logErrorSpy: sinon.SinonSpy;
+  let logInfoSpy: sinon.SinonSpy;
+
+  this.beforeEach(() => {
+    const loggingService = new LoggingService();
+    logErrorSpy = sinon.spy(loggingService, "logError");
+    logInfoSpy = sinon.spy(loggingService, "logInfo");
+    const notificationService = new NotificationService(loggingService);
+
+    moduleResolver = new ModuleResolver(loggingService, notificationService);
+  });
+
+  suite("getPrettierInstance", () => {
+    test("it returns the bundled version of Prettier if local isn't found", () => {
+      const fileName = path.join(
+        getWorkspaceFolderUri("no-dep").fsPath,
+        "index.js"
+      );
+      const prettierInstance = moduleResolver.getPrettierInstance(fileName, {
+        showNotifications: true,
```

Comment:
Needing this preference seems a bit odd to me. It looks like it typically only relates to notifications, but in this case it prevents the log from showing. Is this intentional?

https://github.com/prettier/prettier-vscode/blob/ca5ea1d8dc2bb403db3be252c312801f8b7efbb2/src/ModuleResolver.ts#L117-L119